### DISCUSSION TITLE
Functionality Questions - Budgets, Transfers, Categories

### DISCUSSION THREAD

**Discussion by pjwarner** [firefly-iii/firefly-iii#9142] [Firefly III] [answered]
I know this is kind of complicated and I am trying to articulate it the best I can so here goes.  So I am trying to make sure that I am using the system correctly.  I have all my accounts loaded in, and one thing that I am finding difficult is managing budgets, transfers, and categories.

I use the budgets screen "**Transactions without a budget**" to make sure that all my transactions get linked to budgets, and ones that are one off or not really "budget friendly" I use Categories. (Vacations, Kids school spending etc.)

Is there a way to filter out all transactions that have a budget assigned and/or a category?

Further when I import transactions from my bank(s)/credit cards there is often a need to link these.  Right now I choose one and convert it to a transfer and then delete the transaction in the other account...but I can see where this might cause some problems in the future.  What is the best way to handle transfers between accounts that you are importing?

**Comment by JC5** [comment, marked as answer]
So I see a few questions. What you do is pretty much correct, so no comments there. If you want to find transactions you still need to do, use the page you found already or search for `has_no_category:true` and/or `has_no_budget:true`.

When you import, Firefly III can automatically create transfers if both the source and the destination account are recognized, possibly by mapping the account names through the data importer UI. What you describe works just as well (using a rule) although I use a webhook myself for more complicated changes.

**Reply by pjwarner** [reply]
I will just continue how I am doing the transfers now.

As for the transactions the search functionality is perfect thanks!
